src/Chapter1.hs

```diff
@@ -542,7 +543,12 @@ value after "=" where the condition is true.
 Casual reminder about adding top-level type signatures for all functions :)
 -}
 
-mid x y z = error "mid: not implemented!"
+mid :: (Ord a, Num a) => a -> a -> a -> a
+mid x y z = x + y + z - minim - maxim
+  where
+    minim = min x $ min y z
+    maxim = max x $ max y z
```

Comment:
That's a very clever implementation! :brain: 
Though, I wonder, will it still work around big numbers because of integer overflow :thinking: 
We don't test such cases in our course, but something to keep in mind when implementing it in real life :slightly_smiling_face: